templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -284,6 +284,12 @@ contents:
         iface_type=802-3-ethernet
       fi
 
+      # TODO: Should this persist all ethtool options?
+      ethtool_offload_opts=$(nmcli --get-values ethtool.feature-esp-tx-csum-hw-offload conn show ${old_conn})
+      if [ -n "$ethtool_offload_opts" ]; then
+        extra_phys_args+=(ethtool.feature-esp-tx-csum-hw-offload "${ethtool_offload_opts}" )
```

Comment:
nit: it feels unnecessary but I would just follow the trend and leave that extra space at the beginning

Reply:
Oops, yes. I meant to do that. Don't want to accidentally trigger some weird bash behavior. :-)